pkg/apis/flagger/v1beta1/canary.go

```diff
@@ -294,7 +294,8 @@ type SessionAffinity struct {
 	// ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#attributes
 	// The default value is 86,400 seconds, i.e. a day.
 	// +optional
-	MaxAge int `json:"maxAge,omitempty"`
+	MaxAge            int    `json:"maxAge,omitempty"`
+	PrimaryCookieName string `json:"primaryCookieName,omitempty"`
```

Comment:
We're missing `// +optional` here. Can you also add comments for MaxAge and PrimaryCookieName.